src/tools/wasm-link.cc

```diff
@@ -68,53 +53,26 @@ struct Context {
   ssize_t current_section_payload_offset = 0;
 };
 
-static void on_option(struct OptionParser* parser,
-                      struct Option* option,
-                      const char* argument) {
-  switch (option->id) {
-    case FLAG_DEBUG:
-      s_debug = true;
-      s_log_stream = FileStream::CreateStdout();
-      break;
-
-    case FLAG_OUTPUT:
-      s_outfile = argument;
-      break;
-
-    case FLAG_RELOCATABLE:
-      s_relocatable = true;
-      break;
-
-    case FLAG_HELP:
-      print_help(parser, PROGRAM_NAME);
-      exit(0);
-      break;
```

Comment:
Call me old fashioned by I actually prefer this old syntax.. i find it less noisy.   Don't let that block this change though.